scripts/build/Platform/Windows/build_config.json

```diff
@@ -1,4 +1,32 @@
 {
+  "public_metrics_pipe": {
+    "TAGS": [
+      "periodic-incremental-daily-internal"
+    ],
+    "steps": [
+      "generate_metrics",
+      "upload_metrics"
+    ]
+  },
+  "generate_metrics": {
+    "TAGS": [
+      "periodic-incremental-daily-internal"
```

Comment:
You only need to add the tag to the pipe config if they are meant to run together, otherwise they will run separately.